geomesa/src/main/scala/geotrellis/geomesa/geotools/FeatureToGeoMesaSimpleFeatureMethods.scala

```diff
@@ -0,0 +1,11 @@
+package geotrellis.geomesa.geotools
+
+import geotrellis.proj4.CRS
+import geotrellis.util.MethodExtensions
+import geotrellis.vector.{Feature, Geometry}
+import org.opengis.feature.simple.SimpleFeature
+
+trait FeatureToGeoMesaSimpleFeatureMethods[G <: Geometry, T] extends MethodExtensions[Feature[G, T]] {
+  def toSimpleFeature(featureName: String, featureId: Option[String] = Some(null), crs: Option[CRS] = None)(implicit transmute: T => Seq[(String, Any)]): SimpleFeature =
```

Comment:
Looks like it's just the new review functionality for github - it no longer removes comments when a new change comes in that addresses the issue. Which makes it harder to see if issues are addressed, which is pretty crappy of GitHub.